client/blocks/plugins-update-manager/index.tsx

```diff
@@ -38,15 +50,16 @@ export const PluginsUpdateManager = ( props: Props ) => {
 					</Button>
 				) }
 			</NavigationHeader>
-
-			{ context === 'list' && (
-				<ScheduleList
-					siteSlug={ siteSlug }
-					onNavBack={ onNavBack }
-					onCreateNewSchedule={ onCreateNewSchedule }
-				/>
-			) }
-			{ context === 'create' && <ScheduleCreate siteSlug={ siteSlug } onNavBack={ onNavBack } /> }
+			<ScheduledUpdatesGate needsUpgrade={ ! isEligibleForFeature }>
```

Comment:
```suggestion
			<ScheduledUpdatesGate needsUpgrade={ ! hasScheduledUpdate }>
```

Should this be `hasScheduledUpdate`? Otherwise Simple Sites on a Creator plan would still see the upgrade banner.

Reply:
I think I'll have to change it to another way, so I put the PR back to draft PR. Basically, we might want:
- **Case 1 Users don't have a plan: show the upsell nudge**
- **Case 2 Users with the Creator plan but haven't transferred to Atomic:** We need to ask them to transfer first. I've trying to find a better way to handle case 2 here.